server/plugin_release.go

```diff
@@ -162,7 +162,11 @@ func createTag(ctx context.Context, client *GithubClient, owner, repository, tag
 		var ref *github.Reference
 		ref, _, err = client.Git.GetRef(ctx, owner, repository, "heads/master")
 		if err != nil {
-			return errors.Wrap(err, "failed to get github ref")
+			// perhaps the repo uses main instead?
```

Comment:
Would it make sense to query `default_branch` directly instead? I realize there are likely just two cases we care about, so I'm 0/5, but it looks like the default branch is available from https://docs.github.com/en/rest/reference/repos#get-a-repository.

Reply:
Yep, doable. Thanks for the suggestion!